study-builder/studies/basil/i18n/en.conf

```diff
@@ -7,4 +7,16 @@
     "join_option_mailing_list": "I want to join the mailing list",
     "join_hint_req": "Please choose an option above",
   },
+  "autocomplete": {
+    "name": "Picklist Autocomplete",
+    "title": "Title: Picklist Autocomplete feature",
+  },
+  "country": {
+    "afghanistan": "Afghanistan",
+    "albania": "Albania",
+    "algeria": "Algeria",
+    "american_samoa": "American Samoa",
```

Comment:
Since there's a lot of countries, and since it's just the `basil` demo study, I think we can skip translations. Just use the English for the country names and be done with it, e.g. just hardcode
```
{
   ... [
    { "language": "en", "text": "Canada" },
    { "language": "es", "text": "Canada" },
   ]

   ... [
    { "language": "en", "text": "..." },
    { "language": "es", "text": "..." },
   ]
}
```

Reply:
I generated this conf with StringTemplate, therefore not a big problem to generate 2 languages.